broadcast/DeployPermissionSplitterProxy.s.sol/1/dry-run/run-1706197795.json

```diff
@@ -0,0 +1,97 @@
+{
```

Comment:
Is this the real deployment? If so, why is it labeled "dry-run"?

Reply:
I believe this is the simulation step that happens for [forge scripts](https://book.getfoundry.sh/tutorials/solidity-scripting#high-level-overview)